graphql_compiler/query_planning_and_execution/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright 2021-present Kensho Technologies, LLC.
```

Comment:
This module name is a bit of a mouthful :) Would it be worth trying to come up with something shorter?